public/New-DbaConnectionString.ps1

```diff
@@ -196,7 +196,6 @@ function New-DbaConnectionString {
         [string]$ClientName = "custom connection",
         [int]$ConnectTimeout,
         [string]$Database,
-        [ValidateSet('Mandatory', 'Optional', 'Strict', 'True', 'False')]
         [switch]$EncryptConnection = (Get-DbatoolsConfigValue -FullName 'sql.connection.encrypt'),
```

Comment:
oh man, wait this is the problem 🤔

Reply:
No, that is not a (technical) problem. When I refactored the command I did not remove this line by acident. But as it is a switch parameter, it does not have any effect. It's just a useless line. But if you look at the code and overlook the `[switch]` you may think that you have the choice here.